geomstats/_backend/pytorch/__init__.py

```diff
@@ -354,7 +358,7 @@ def sqrt(x):
     return torch.sqrt(x)
 
 
-def isclose(x, y, rtol=1e-5, atol=1e-8):
+def isclose(x, y, rtol=1e-5, atol=atol):
```

Comment:
Would it make sense to have `rtol=rtol` there? @Maya95assal @nguigs

Reply:
(Same question for everywhere else in the PR)

Reply:
Yes I think we should have a custom value for rtol according to the backend, because the threshold is in fact atol + b * rtol, so if b >> atol, and rtol is not so small, rtol will dominate. See https://numpy.org/doc/stable/reference/generated/numpy.isclose.html
```python
>>> import geomstats.backend as gs # numpy backend here
>>> gs.isclose(gs.array([1. + 1e-5, 1. - 1e-9]), 1., atol=1e-12)
array([ True,  True])
```